frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -98,10 +98,16 @@ object Interpreter {
     val scalaVersion = bloop.internal.build.BuildInfo.scalaVersion
     val zincVersion = bloop.internal.build.BuildInfo.zincVersion
     val developers = bloop.internal.build.BuildInfo.developers.mkString(", ")
+    val javaVersion = Option(System.getProperty("java.version"))
```

Comment:
```scala
sys.props.get("java.version")
sys.props.get("java.home")
```
Should get an option and noe need to pack it in one here.

Reply:
I actually prefer this over using the Scala API :smile:

Reply:
Haha, fine with me. Just thought it might be a bit shorter.